gfxr_ext/decode/dive_vulkan_replay_consumer.cpp

```diff
@@ -699,5 +699,16 @@ void DiveVulkanReplayConsumer::ProcessFrameEndMarker(uint64_t frame_number)
     }
 }
 
+void DiveVulkanReplayConsumer::Process_vkGetFenceFdKHR(
+const ApiCallInfo&                                 call_info,
+VkResult                                           returnValue,
+format::HandleId                                   device,
+StructPointerDecoder<Decoded_VkFenceGetFdInfoKHR>* pGetFdInfo,
+PointerDecoder<int>*                               pFd)
+{
+    // Workaround to replay compositor captures.
```

Comment:
can you add more comment on why this func needs to be skipped for compositor? thanks